File: ass53/Program4.java
/*
4. Write Java program which accept N numbers from user and display all such elements which are divisible by 3 and 5. 
Input :  N :   6 
  Elements : 85 66 3 15 93 88  
Output : 15 
*/
import java.lang.*;
import java.util.*;
class ArrayDemo
{
    void Even(int arr[])
    {
        int iSum1 = 0;
        for(int i=0; i<arr.length; i++)
        {
            if((arr[i]%3==0)&&(arr[i]%5==0))
            {
               
               System.out.println("Divisible 5 nad 3 is : " +arr[i]);
            }   
        }
    }
}
class Program4
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the Element");
        int size = sobj.nextInt();
        int arr[] = new int[size];
        System.out.println("Enter the Array elements");
        for(int i=0; i<arr.length; i++)
        {
            arr[i] = sobj.nextInt();
        } 
        ArrayDemo dobj = new ArrayDemo();
        dobj.Even(arr);
    }
}

File: ass24/program4.c
/*
4. Accept N numbers from user and display all such numbers which contains  3 digits in it. 
Input :  N :   6 
  Elements : 8225  665 3 76 953 858  
Output : 665 953 858
*/
#include<stdio.h>
#include<stdlib.h>
int LastOccurance(int arr[],int iSize)
{
    int iCnt=0;
    for(iCnt=0;iCnt<iSize;iCnt++)
    {
        if((arr[iCnt]>=100)&&(arr[iCnt]<=999))
        {
            printf("%d",arr[iCnt]);
        
        }
        
    }
    
}
int main()
{
   int iValue=0, iN=0,iRet=0,iCnt=0;
    int *p=NULL;
    printf("enter the input\n");
    scanf("%d",&iValue);
    p=(int*)malloc(sizeof(int)*iValue);
    if(p==NULL)
    {
        return -1;
    }
    for(iCnt=0;iCnt<iValue;iCnt++)
    {
        scanf("%d",&p[iCnt]);
    }
    
    LastOccurance(p,iValue);
    
}